Functions/GeoTrack.ps1

```diff
@@ -101,6 +101,25 @@ Function Convert-GeoTrack {
     }
 }
 
+Function Get-GoogleSessionVariable {
+    [CmdletBinding()] param(
+        [string] $email,

```

Comment:
email/password would preferably be a PSCredential which you can retreive with a call to Get-Credential.